Documentation~/Blender.md

```diff
@@ -112,7 +112,16 @@ MeshSync has functionality to create baked material copies for each object to al
 | Bake to individual materials | Performs the bake. |
 | Restore original materials | Removes the baked material copies and assigns the original materials back to all objects in the scene. |
 | Baked texture path | Folder to save baked textures in. |
-| Baked texture size | Baked texture dimensions. |
+
+|**Texture dimension options** |**Description** |
+|:---     |:---|
+| Texture dimensions | Selects how texture dimensions are set. |
+| Pixels | Set exact texture dimensions |
+| Baked texture size | Size of the baked texture in pixels. |
+| Texel density | Set texture dimensions based on a texel density. |
+| Texels / World Unit | How many pixels the baked texture should have relative to the area of the polygon in blender's world space. The average of each polygon's area in the UV map vs its world space area is used to calculate this. |
+| Max texture size | Limits the texture dimensions so they don't get too large for higher texel densities. |
```

Comment:
Can "Pixels" and "Baked" be extracted to another table that describes the values of "Texture dimensions"?

Reply:
I've changed that to be consistent with the UV settings documentation now.